element_lab/lab.py

```diff
@@ -32,24 +42,45 @@ class Lab(dj.Lookup):
 
 @schema
 class Location(dj.Lookup):
+    """Location of animal housing or experimental rigs.
+
+    Attributes:
+        Lab (foreign key): Lab key.
+        location ( varchar(32) ): Location of the lab.
+        location_description ( varchar(255) ): Optional. Description of the lab location.
```

Comment:
This table permits a many-to-one location with the lab, so I think I'd rewrite to suggest multiple. Please also update the description with comments, including a `# Heading comment` after the opening `"""`
```suggestion
    """Location of research (e.g., animal housing or experimental rigs)

    Attributes:
        Lab (foreign key): Lab key.
        location ( varchar(32) ): Location of a space related to the lab
        location_description ( varchar(255), optional ): Description of the location.
```